extensions/pi-logger/CHANGELOG.md

```
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.2.0] - 2026-05-08
+
+### Changed
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)